business/apps.go

```diff
@@ -151,7 +152,15 @@ func fetchNamespaceApps(k8s kubernetes.IstioClientInterface, namespace string, a
 	go func() {
 		defer wg.Done()
 		var err error
-		services, err = k8s.GetServices(namespace, nil)
+		// Check if namespace is cached
+		if kialiCache != nil && kialiCache.CheckNamespace(namespace) {
+			// Cache uses Kiali ServiceAccount, check if user can access to the namespace
+			if _, err := layer.Namespace.GetNamespace(namespace); err == nil {
```

Comment:
I guess it should be simple equal `=` instead of `:=`, no? Else the variable is shadowed and the error will not be reported on chanel